services/vcpkg/vcpkg-version.service.js

```diff
@@ -4,9 +4,20 @@ import { fetchJsonFromRepo } from '../github/github-common-fetch.js'
 import { renderVersionBadge } from '../version.js'
 import { NotFound } from '../index.js'
 
-const vcpkgManifestSchema = Joi.object({
-  version: Joi.string().required(),
-}).required()
```

Comment:
Can add a comment here linking to https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json?source=recommendations#version to make it a bit more obvious what is going on here

Reply:
That's a good idea to directly link the documentation! Add this in  https://github.com/badges/shields/pull/8945/commits/79d48e64e05ca0e70a8a2ac3b305b54178b5989e